### ISSUE TITLE
Issue with redirects when redirecting link is blocked

### ISSUE THREAD

**Issue by vertigo220** [sblask/webextension-skip-redirect#94]
### New Issue Checklist (tick off like this: `[x]`)

- [x] I checked existing issues (open and closed) for possible duplicates
- [x] I can reproduce the problem on the latest stable version (not nightly!)
- [x] The issue is not about adding entries to the default blacklist

### What is your browser?

- [x] Waterfox 56.1.0x64

### What is your operating system?

- [x] Windows

### Description (please include examples/screenshots where applicable)

Clicking this link (edited since it _is_ a tracking link tied to me):

http://www.jdoqocy.com/click-6302448-13218011?URL=https%3A%2F%2Fwww.fanatical.com%2Fen%2Fgame%2Fbatman-arkham-origins

gives me this image:

![image](https://user-images.githubusercontent.com/12024557/39833753-032581d4-5399-11e8-9654-f04141ad042e.png)

because jdoqocy is blocked by uMatrix. I can simply click the second link (the fanatical one) to get there, but I wonder if Skip Redirect could get by this by and keep uMatrix from even having to block it.

Edit: Forgot to mention, and I'm not sure if this is helpful for or even applicable to this issue, but this link:

https://slickdeals.net/?adobeRef=74eac2fe4be511e8821f668ace3e36770009&pno=384839&lno=1&sdtid=11534351&trd=Adorama&afsrc=1

isn't blocked, and Skip Redirect sends me to the appropriate page, even though with SR disabled uMatrix blocks the link due to ojrq.net being blocked.

**Comment by sblask** [comment]
The extension has no influence over the order of events it gets. Sometimes it might get an event first, another time it might be a different extension. The order might be consistent too, but only the browser knows the internals of how this is handled. Nothing I can do.
